xbmc/video/VideoDatabase.cpp

```diff
@@ -2097,15 +2097,19 @@ int CVideoDatabase::SetDetailsForMovie(const std::string& strFilenameAndPath, co
     }
     // update our movie table (we know it was added already above)
     // and insert the new row
-    std::string sql = "update movie set " + GetValueString(details, VIDEODB_ID_MIN, VIDEODB_ID_MAX, DbMovieOffsets);
+    std::string sql = "UPDATE movie SET " + GetValueString(details, VIDEODB_ID_MIN, VIDEODB_ID_MAX, DbMovieOffsets);
     if (idSet > 0)
```

Comment:
I tried to do something like:

```
std::string CVideoDatabase::AddUpdateParameter(std::string name, int value)
{
  if (value > 0)
        return PrepareSQL(", %s = %i", name, value);
      else
        return StringUtils::Format(", %s = NULL", name);
}
```
And then calling doing ```sql += AddUpdateParameter("idSet", idSet);``` or sql += AddUpdateParameter("userrating", details->m_iUserRating);

But for some reason the call gets lost in the PrepareSQL part :(